mirgecom/__init__.py

```diff
@@ -25,26 +25,23 @@
 import sys
 from warnings import warn
 
-try:
-    from mpi4py import MPI
+from mpi4py import MPI
 
-    # This code avoids slow startups due to file system locking when running with
-    # large numbers of ranks. See https://github.com/illinois-ceesd/planning/issues/27
-    # for details.
+# This code avoids slow startups due to file system locking when running with
+# large numbers of ranks. See https://github.com/illinois-ceesd/planning/issues/27
+# for details.
 
-    size = MPI.COMM_WORLD.Get_size()
+size = MPI.COMM_WORLD.Get_size()
 
-    if size >= 128:
-        for mod in ["pyopencl", "pytools", "loopy"]:
-            if mod in sys.modules:
-                warn("{} already loaded, not adjusting XDG_CACHE_HOME.".format(mod))
+if size >= 128:
+    for mod in ["pyopencl", "pytools", "loopy"]:
+        if mod in sys.modules:
+            warn("{} already loaded, not adjusting XDG_CACHE_HOME.".format(mod))
```

Comment:
This warning is too opaque. Write for a  hapless student who might have no idea what any of it means, but whose runs are dog-slow.

Reply:
Made the warning more verbose.